packages/xdl/src/detach/IosNSBundle.js

```diff
@@ -257,7 +257,17 @@ async function _configureInfoPlistAsync(context: StandaloneContext) {
 
     // app name
     infoPlist.CFBundleName = config.name;
-    infoPlist.CFBundleDisplayName = config.name;
+
+    if (
+      context.build &&
+      context.build.isExpoClientBuild &&
+      context.build.isExpoClientBuild() === 'client'
```

Comment:
`isExpoClientBuild()` sounds like it should return a boolean

Reply:
1) Ville is right, it returns a boolean - https://github.com/expo/expo-cli/blob/master/packages/xdl/src/detach/StandaloneBuildFlags.js#L42
2) You don't need to check if `context.build` and `context.build.isExpoClientBuild` exist, they always exist.
3) The following line should be enough:
```infoPlist.CFBundleDisplayName = context.build.isExpoClientBuild() ? 'Expo' : config.name;```